src/gpu/intel/compute/kernel_ctx.hpp

```diff
@@ -85,6 +85,13 @@ class kernel_ctx_t {
     // operations when native 64-bit operations are unsupported.
     void use_int32_offset(bool value) { use_int32_offset_ = value; }
 
+    void require_stateless_addressing(bool value) {
+        require_stateless_addressing_ = value;
+    }
+    bool require_stateless_addressing() const {
+        return require_stateless_addressing_;
+    }
```

Comment:
This is a more traditional convention.
```suggestion
    void set_stateless_addressing(bool value) {
        stateless_addressing_ = value;
    }
    bool get_stateless_addressing() const {
        return stateless_addressing_;
    }
```

Reply:
Yes, it just did not make sense from language point of view. I assumed `require_stateless_addressing` is more descriptive of what we are trying to get/set.